cmd/influx/query.go

```diff
@@ -4,9 +4,13 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/influxdata/flux"
+
 	"github.com/influxdata/flux/repl"
+	_ "github.com/influxdata/flux/stdlib" // Import the stdlib
 	platform "github.com/influxdata/influxdb"
-	_ "github.com/influxdata/influxdb/query/builtin"
+
+	_ "github.com/influxdata/influxdb/query/stdlib" // Import the stdlib
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
```

Comment:
Can you remove the extra blank lines here and below?

Reply:
yeah, no problem.